diff --git a/srms2pdf b/srms2pdf
index bfdb2d9..da5f57c 100755
--- a/srms2pdf
+++ b/srms2pdf
@@ -11,4 +11,3 @@ done
 gs -q -sDEVICE=pdfwrite -sOutputFile=$FILE.pdf -dNOPAUSE -dBATCH $FILE*.ps
 rm $FILE*.fig
 rm $FILE*.ps
-open $FILE.pdf
diff --git a/srms2png b/srms2png
index 158e7e8..1284b2c 100755
--- a/srms2png
+++ b/srms2png
@@ -10,4 +10,3 @@ do
 done
 rm $FILE*.fig
 rm $FILE*.pdf
-open $FILE*.png
